flask_appbuilder/security/manager.py

```diff
@@ -866,6 +903,11 @@ def auth_user_ldap(self, username, password):
         """
         if username is None or username == "":
             return None
+        else:
```

Comment:
likewise you can remove the else and unindent the following lines, because the if above results in a return statement